test/metabase/query_processor_test/advanced_math_test.clj

```diff
@@ -14,11 +13,9 @@
         :order-by    [[:asc [:field (mt/id :venues :id) nil]]]
         :limit       1}
        (mt/run-mbql-query venues)
-       mt/rows
-       ffirst
-       double
        ;; Round to prevent minute differences across DBs due to differences in how float point math is handled
-       (u/round-to-decimals 2)))
+       (mt/formatted-rows [2.0])
```

Comment:
nit, it's weird that this looks for `float` to represent precision, I believe the impl could be changed to `number?` safely.

Reply:
I'm not 100% sure I'm following, `(mt/formatted-rows [2.0] ...)` has been our shorthand for round-to-2-decimal-places since basically forever, the old code was doing exactly the same thing but not using our normal pattern. The old code actually failed with an NPE for me because I messed up test data loading and it was trying to call `(u/round-to-decimals 2 nil)`... that's why I switched it to do the same thing with `u/formatted-rows`, because it has built-in checks for `nil`.

We need the rounding here because otherwise some DBs will return like `1.23456` and others will return `1.23455` and those are not meaningful differences